boards/arm/arduino_due/Kconfig.defconfig

```diff
@@ -40,20 +40,15 @@ endif # GPIO
 
 if I2C
 
-config I2C_ATMEL_SAM3
+config I2C_SAM_TWI
```

Comment:
Please add a comment in the commit message to note that you're changing the default I2C driver for this board

Reply:
Done.